### PR TITLE
Add .NET Core site

### PR DESCRIPTION
**Pull request by blackdwarf**
This is a big change. We've added the Jekyll-powered .NET Core site as a sub-folder to dotnet's main site. The changes are mostly isolated, except adding a few links from the main page of the dotnet org site. 

You can see the site running at http://blackdwarf.github.io/dotnet.github.io/ and http://blackdwarf.github.io/dotnet.github.io/core/ for core site. 

/cc @richlander @martinwoodward @BethMassi

### CODE REVIEW

**Comment by thesmithee** [comment]
Oh, and would be great if someone (@richlander??) could add me as a contributor to this site if this PR is accepted. :)

**Comment by BethMassi** [comment]
@blackdwarf This is absolutely awesome :+1:

**Comment by thesmithee** [comment]
Thanks @martinwoodward! :)

**Comment by BethMassi** [comment]
@blackdwarf LOL. Are you suggesting I speak for @martinwoodward ? Only sometimes I guess :smiley:

**Comment by martinwoodward** [comment]
I think @blackdwarf is referring to the fact I just gave him commit rights ;-)

+1 to the change - very awesome. You don't fancy refreshing the dotnet.github.io site a little to fit in with that look and feel do you as well?  Come to think of it, if you want to help me with the dotnetfoundation.org refresh as well you are more than welcome!

Anyway, I'll let @richlander press the merge button as it's his project - but I love it.

**Comment by thesmithee** [comment]
Martin has it right, sorry for not being clear Beth. :) Thanks for the comments guys. 

I have no problems helping you @martinwoodward with the refresh. We should open an issue to track that work and see what spells we can weave. :)

**Comment by BethMassi** [comment]
When can we merge this and point people at it? Is it ready for eyeballs? I have a post for O'Reilly and one of the calls to action is to check out .NET Core. The getting started here looks fantastic and would love to link to it.

**Comment by KimTapia** [comment]
Let me know if I can help with design/graphics/hTML/CSS.  I'm working on the Foundation site today.

**Comment by thesmithee** [comment]
As soon as @richlander LGTMs this I can merge and it should be live in 2-5 minutes post that. you can go through the content in the files in the commits and see if there is anything completely out of wack there. 

@KimTapia thanks. I mainly used the CSS styles from the dotnet main site. The only differences are the fonts (Roboto and Open Sans instead of Segoe).

File: _includes/core/footer.html [hunk 1]
```diff
@@ -0,0 +1,20 @@
+<footer>
+  <div class="inner-footer">
+    <p>© 2015 Microsoft, Inc.</p>
```
Comment (martinwoodward) [inline _includes/core/footer.html]: Should be (C) .NET Foundation (no year).  FYI, recommended copyright under MSFT would be (C) Microsoft Corp. (again no year and definitely not Microsoft, Inc).

**Comment by martinwoodward** [comment]
Also - doesn't need to be in this round of changes, but be great to tighten up on the console animated gif so that it doesn't have stuff around the chrome of the console window. Even better would be to use an Ubuntu console rather than a Putty on Windows.

**Comment by thesmithee** [comment]
Well, this was an Azure VM. :) But agreed @martinwoodward , would be great to have the actual OS there.

File: _includes/core/footer.html [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    <p>
+      <a href="https://twitter.com/microsoft" target="_blank"><span class="fa fa-stack fa-lg">
```
Comment (martinwoodward) [inline _includes/core/footer.html]: Can we link to dotnetfdn please?

File: _includes/core/footer.html [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+        <span class="fa fa-square-o fa-stack-2x"></span>
+        <span class="fa fa-twitter fa-stack-1x"></span>
+      </span></a>
+      <a href="https://www.github.com/microsoft" target="_blank"><span class="fa fa-stack fa-lg">
```
Comment (martinwoodward) [inline _includes/core/footer.html]: Can we link to http://github.com/dotnet - also can the GitHub link come before the twitter link?

File: _includes/core/footer.html [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+        <span class="fa fa-square-o fa-stack-2x"></span>
+        <span class="fa fa-github fa-stack-1x"></span>
+      </span></a>
+      <a href="https://www.facebook.com/Microsoft" target="_blank"><span class="fa fa-stack fa-lg">
```
Comment (martinwoodward) [inline _includes/core/footer.html:13]: Well - that's a good point.  I should probably set up a proper Facebook presence for the .NET Foundation...

File: _includes/core/hero.html [hunk 5]
```diff
@@ -0,0 +1,12 @@
+<div class="container hero">
+
+  <div class="starter-template">
+    <h1 class="dotnet">.NET Core</h1>
+    <p class="lead">A general purpose managed framework</p>
```
Comment (martinwoodward) [inline _includes/core/hero.html:5]: Not in love with this tagline.

File: _includes/core/value_props.html [hunk 6]
```diff
@@ -0,0 +1,63 @@
+<div id="vp-container">
+  <div id="initial-vps">
+    <div class="row value-props">
+      <div class="col-lg-4 text-center animated fadeInLeft">
+        <span class="fa fa-cubes fa-3x"></span>
+        <h2 class="value-prop-heading">Modular</h2>
+        <p>.NET Core is a set of runtime, library and compiler components. Microsoft uses these components in various configurations for device and cloud workloads. You can do the same for your app or service.</p>
```
Comment (martinwoodward) [inline _includes/core/value_props.html:7]: Would be great if we can talk about more companies than just Microsoft. Maybe not possible yet for .NET Core?

File: _includes/core/value_props.html [hunk 7]
```diff
@@ -0,0 +1,63 @@
...
+        <p>You can create .NET Core apps that run on multiple OSes and CPUs. .NET Core runs on Windows. Ports are in progress for Linux, OS X and FreeBSD, as is integration with the LLVM compiler.</p>
+      </div><!-- /.col-lg-4 -->
+    </div><!-- /.row -->
+  </div>
+  <div id="rest-vps">
+    <div class="row value-props">
+      <div class="col-lg-4 text-center">
+        <span class="fa fa-magic fa-3x"></span>
+        <h2 class="value-prop-heading">Modern</h2>
+        <p>.NET Core brings with it a set of languages, led  by C# with support for modern language features, like generics, Language Integrated Query (LINQ), async support and more. It is backed by a managed compiler called "Roslyn" that is exposed to the runtime, usable as a service.</p>
```
Comment (martinwoodward) [inline _includes/core/value_props.html]: Is it worth highlighting the languages that you can write with against .NET Core. I get asked about VB and F# as well as C#

**Comment by martinwoodward** [comment]
After this is merged, would you be able to send a PR to update the [NetCore5.md](https://github.com/dotnet/home/blob/master/projects/NETCore5.md) file to point at the new homepage for .NET Core?  (Also we probably want to rename the file netcore rather than NetCore5 so will also need to rev the [projects.json](https://github.com/dotnet/home/blob/master/projects/projects.json) file...)

Those files control these pages:
- http://www.dotnetfoundation.org/projects
- http://www.dotnetfoundation.org/netcore5

File: _includes/core/value_props.html (hunk 7 above)
Comment (thesmithee) [inline _includes/core/value_props.html]: Good suggestion, this is good feedback.

File: _includes/core/footer.html (hunk 3 above)
Comment (thesmithee) [inline _includes/core/footer.html]: Sure, will be done.

**Comment by thesmithee** [comment]
@martinwoodward sure, can I bother you to open up an Issue and assign to me? Would help me track it post-merge. Thanks!

**Comment by thesmithee** [comment]
Sorry about this, had to rebase to add a proper set of name/email pairs so that commits can be seen as me instead of this other blackdwarf dude. :)

**Comment by richlander** [comment]
LGTM. Thanks for building this.

In future, try to add a trailing line to your files in GH. That's what those funny red arrows are about in the "files" view.

**Pull request merged**